src/backend/auth/routes.py

```diff
@@ -0,0 +1,31 @@
+import identity
+from flask import current_app, redirect, render_template, request, session, url_for
+
+from backend.auth import bp
```

Comment:
FYI, `identity` 0.7+ provides higher level API tailored to Flask, so you probably do not need this file anymore.